### ISSUE TITLE
Review agile position descriptions for content updates

### ISSUE THREAD

**Issue by jstrothman** [18F/guides#313]
The Derisking Guide links to [brief agile position descriptions](https://github.com/18F/agile_position_descriptions) which last had substantive updates in Jan 2020 (aside from the addition of the Product Manager role in Feb 2021).

These could use SME review for currency (e.g. programming languages) and equity lens. 
Also per Shea B (Cloud) noted that we should add roles for Cloud Solutions Architect or Cloud Engineer.

**Comment by bpdesigns** [comment]
@cantsin should the derisking guide include cloud roles?
@elisaachen should there be a service design role as part of an agile team?

**Comment by cantsin** [comment]
I agree that these roles need SME review. As for adding, we (18F) seldom do cloud solutions or devsecops work so we are not best suited to write position descriptions for these roles.

**Comment by elisaachen** [comment]
@bpdesigns  - can you point me to where on our derisking guide this is? Was trying to understand the context where this content lies.

**Comment by ameliaswong** [comment]
No @bpdesigns , the updates to the DRG don't include agile position descriptions. Gave you and @elisaachen viewer access to the [document](https://docs.google.com/document/d/1bJ2duMLuAdEaSpHsHlFvlL2Jq9xEvuGS3ZLZ-_lc4gQ/edit#heading=h.itayxhkc9kqn) so you can review content if helpful (it's going through TTS approvals which is why I'm being stingy with permissions). 

Also FYI, I'm writing up a deployment strategy after talking with @adunkman and @peterrowland. I'll share that with you both when we've gotten a full draft.
